index.js

```diff
@@ -139,8 +139,15 @@ module.exports = function(version, _options) {
             }
             name = name.replace(' (' + step.ref + ')', '');
 
-            if (name && ref && name !== ref) {
+            // In attempt to avoid using the ceremonial name of a way,
```

Comment:
I’m not sure where the term “ceremonial name” comes from, but this PR appears to handle two common scenarios:

* Plain old road names assigned to freeways, like “Northeast Expressway”
* Memorial highway designations, like “Martin Luther King Junior Memorial Highway” or “CHP Officer David W. Manning Memorial Freeway”

Let’s use the term “highway name” to refer to both senarios.